recipes/djinni-support-lib/1.x.x/conanfile.py

```diff
@@ -102,6 +102,9 @@ def validate(self):
             if self._objc_support or self._jni_support or self._python_support:
                 raise ConanInvalidConfiguration(
                     "C++/CLI is not yet supported with other languages enabled as well. Disable 'with_jni', 'with_objc' and 'with_python' options for a valid configuration.")
+        if self._python_support:
+            if self.settings.os == "Windows":
+                raise ConanInvalidConfiguration("Python on Windowns is not fully yet supported, please see https://github.com/cross-language-cpp/djinni-support-lib/issues.")
```

Comment:
```suggestion
                raise ConanInvalidConfiguration("Python on Windows is not fully yet supported, please see https://github.com/cross-language-cpp/djinni-support-lib/issues.")
```

You have a typo. Commit this change and the bot will rebuild and then we'll see.